src/game/gui/spriteimage.h

```diff
@@ -5,5 +5,6 @@
 #include "video/surface.h"
 
 component *spriteimage_create(const surface *img);
+void spriteimage_owns_sprite(component *c, bool owns_sprite);
```

Comment:
spriteimage_set_owns_sprite, if we want to be pedantic. Since that's that the rest of the GUI side uses.

Reply:
you can't spell pedantic without antic